### PR TITLE
Added  login and signup page to The Banking Website

### LINKED ISSUE #2881: Add a Contact us and Login/Signup to the banking website project
**Describe the Feature you want to end**
I would like to add a signup/login and contact page to the Banking website

**Screenshots (optional)**
image

### PR DESCRIPTION
**Pull request by PoulavBhowmick03**
# Title and Issue number 
Add a Contact us and Login/Signup to the banking website project #2881

Close #2881

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

**Comment by PoulavBhowmick03** [comment]
image
image
@MohitGupta121

File: .vscode/settings.json
Comment (MohitGupta121) [inline .vscode/settings.json:1]: Again you change this, please revert this changes.

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

**Comment by PoulavBhowmick03** [comment]
Done @MohitGupta121

**Review by MohitGupta121 — APPROVED** [review]
